app/views/pages/information/includes/_information-side-navigation.njk

```diff
@@ -0,0 +1,19 @@
+{% set trgPages = [
+  { title: "Information home", url: "/information" },
+  { title: "Data", url: "/information/data" },
+  { title: "Platform and Infrastructure", url: "/information/platform-and-infrastructure" },
+  { title: "Interoperability", url: "/information/interop" },
+  { title: "Cloud", url: "/information/cloud" },
+  { title: "Digital & UX", url: "https://service-manual.nhs.uk" },
```

Comment:
This links to the NHS digital service manual, is this correct?

Reply:
This will change once the page from Dean has been created.

Reply:
I am checking this with content authors, I will leave this open for reference.